DirectProgramming/DPC++FPGA/ReferenceDesigns/qrd/README.md

```diff
@@ -40,10 +40,9 @@ The QR decomposition algorithm factors a complex _m_×_n_ matrix, where _m_ ≥
 
```

Comment:
the _m_x_n_ is not rendering as you probably want (the underscores around the x are being displayed).  I suggest adding spaces around the x.